osext/browser_process.go

```diff
@@ -23,20 +25,23 @@ type BrowserProcess struct {
 	process *os.Process
 
 	// Channels for managing termination.
-	lostConnection             chan struct{}
+	LostConnection             chan struct{}
 	processIsGracefullyClosing chan struct{}
-	processDone                chan struct{}
+	ProcessDone                chan struct{}
 
 	// Browser's WebSocket URL to speak CDP
 	wsURL string
 
 	// The directory where user data for the browser is stored.
-	userDataDir *storage.Dir
+	UserDataDir *storage.Dir
 
 	logger *log.Logger
 }
 
-func NewBrowserProcess(
+// New creates a new osext. It will start
+// a child process where the internet browser will
+// run.
+func New(
```

Comment:
Considering that we have now renamed the pkg to `osext`, should we rename this to `NewBrowserProcess`?

Reply:
Yep, good spot. Resolved in 2cdcab34d24729f1385b9a6660f27232204ab13e